postfix/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:latest
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+    && echo "postfix postfix/main_mailer_type select Internet Site" | debconf-set-selections \
+    && echo "postfix postfix/mailname string mysubdomain.mydomain.com" | debconf-set-selections \
+    && apt-get install -y postfix mailutils inetutils-syslogd locales \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+RUN sed -i 's_^mynetworks = .*$_mynetworks = 0.0.0.0/0 [::]/0_g' /etc/postfix/main.cf
+RUN mkdir -p /var/spool/postfix/etc/ssl && chown -R postfix /var/spool/postfix
+
```

Comment:
Same here you could use the same RUN command in the same line for lines 5, 11 and 12

Reply:
Have your read this guide? https://docs.docker.com/develop/develop-images/dockerfile_best-practices
Having multiple layers is not bad. It is only bad, if you try to delete things in the new layer, as that does not shrink the size.
If you don't believe me, you could use [dive](https://github.com/wagoodman/dive) or [docker-explorer](https://github.com/google/docker-explorer) instead.